src/main/java/com/revature/models/Admin.java

```diff
@@ -20,7 +28,7 @@
 @Component
 @Entity
 @Table(name="admins")
-@NoArgsConstructor @Data
+@Getter @NoArgsConstructor @Setter @EqualsAndHashCode @ToString
```

Comment:
@ Data should be used instead of Setter, Getter, EqualsAndHashCode, and ToString.

Reply:
Resolved issue so that only @Data is used.